Meadowlark-js/backends/meadowlark-postgresql-backend/test/integration/Delete.test.ts

```diff
@@ -76,7 +76,8 @@ describe('given the delete of a non-existent document', () => {
 
   afterAll(async () => {
     await deleteAll(client);
-    await resetSharedClient();
+    client.release();
```

Comment:
Realized that the Postgres deleteAll function has a client.release in it. Would rather not have a release in that function so updated the integration tests to release after calling deleteAll